### ISSUE TITLE
[BUG] after double-clicking on the logo, the session is terminated

### ISSUE THREAD

**Issue by yursh** [Leantime/leantime#2033]
**What set up are you using**
* [x] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**

double-clicking on the logo will break the session. This does not happen every time. sometimes it does, sometimes it doesn't. This behaviour started to happen after the "big update". 

At the same time, you can usually work in the project itself, it rarely breaks.

The configuration specifies a session time of 8 hours as standard.

**To Reproduce**
1. Go to 'Home'
2. Click on 'LOGO'
![Bildschirmfoto 2023-10-25 um 01 05 13](https://github.com/Leantime/leantime/assets/38814372/498c4111-e73d-45c9-bf36-252ef91eb907)
4. Click on 'LOGO'
5. Click on 'LOGO'
6. Click on 'LOGO'
7. You should Login

**Expected behavior**

**Leantime Version**
2.4.4

**Server**
nginx

![Bildschirmfoto 2023-10-25 um 00 46 53](https://github.com/Leantime/leantime/assets/38814372/9662ef3e-af17-49e6-9a3d-75179208b30a)

**PHP / MySQL Version**
PHP Version 8.1

**Comment by marcelfolaron** [comment]
Thanks. Which browser are you using?
We got another report that this is a problem in safari but not in chrome.
Is that the same for you?

**Comment by yursh** [comment]
Hello, thank you for your reply. I use safari browser by default. Yesterday I also tried to use Google chrome and the session was also broken. Today in Google Chrome site works fine and the session is not broken. 

In safari triple click on the logo and you need to log in again. 

I also tried to go to yours cloud Workspace. In the same way, the session is broken after 3 clicks on Logo using Safari.

**Comment by marcelfolaron** [comment]
Thank you. That helps. I will investigate what is going on in Safari.

**Comment by yursh** [comment]
Hi, I use ubuntu server and hosting software. I keep it constantly updated. I looked through the logs in Google Chrome and did not find the 303 redirect. However, as soon as I logged in from safari, I immediately got these two 303s and the login page. I think everything is connected to these 303s. Let me know if I can help you identify the problem.
![Bildschirmfoto 2023-10-25 um 17 29 13](https://github.com/Leantime/leantime/assets/38814372/038dae0f-64a8-42f0-a5db-6c9daf03d5c5)
